Source/UI/NewUI/PaymentForm/View/CardNumber/CardNumberView.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  CardNumberView.swift
+//  Frames
+//
+//  Created by Harry Brown on 05/07/2022.
+//  Copyright © 2022 Checkout. All rights reserved.
+//
+
+import UIKit
+import Checkout
+
+protocol CardNumberViewDelegate: AnyObject {
+  func update(cardNumber: String)
+}
+
+final class CardNumberView: UIView {
+  weak var delegate: CardNumberViewDelegate?
+
+  private let cardUtils = CardUtils()
```

Comment:
@harry-brown-cko to put something in backlog to move cardUtils to checkout iOS SDK as CardFormatter class to have a format method in it as a tech investment story. Reason the CardUtils does not need to be public anymore in Frames and can live inside Checkout iOS SDK. This also needs a update to API spec.